### PR TITLE
Redirect all URLs to readthedocs website

### PR DESCRIPTION
**Pull request by bqth29**

### CODE REVIEW

File: pages/docs/parameters/parameters.md
Comment (pet-mit) [inline pages/docs/parameters/parameters.md:1]: why did you not redirect this?

File: pages/docs/tutorials/run-rao.md
Comment (pet-mit) [inline pages/docs/tutorials/run-rao.md:1]: same

**Review by pet-mit — CHANGES_REQUESTED** [review]

File: pages/docs/parameters/parameters.md
Comment (bqth29) [reply pages/docs/parameters/parameters.md:1]: I did

File: pages/docs/tutorials/run-rao.md
Comment (bqth29) [reply pages/docs/tutorials/run-rao.md:1]: I did too

**Review by pet-mit — APPROVED** [review]

**Pull request merged**